pyproject.toml

```diff
@@ -11,6 +11,9 @@ dependencies = [
     "flask-sqlalchemy>=3.1.1",
     "flask-migrate>=4.0.5",
     "python-dotenv>=1.0.0",
+    "Flask-RQ2>=18.3",
+    "Pillow>=10.1.0",
+    "setuptools>=69.0.3",
```

Comment:
What is it required for?

Reply:
Flask-RQ2 depends on setuptools, but it has not been declared.